crates/red_knot_python_semantic/src/types.rs

```diff
@@ -673,6 +673,36 @@ impl<'db> Type<'db> {
             Type::Tuple(_) => builtins_symbol_ty(db, "tuple"),
         }
     }
+
+    /// Return the string representation of this type when converted to string as it would be
+    /// provided by the builtin `str` function at runtime (see `__str`). If that can't be
+    /// determined, return `None`.
```

Comment:
Nit: calling `str` will involve `Type::call` on the `str` type, so it's clearer if here we only mention the `__str__` builtin, which is what this method really represents.
```suggestion
    /// Return the string representation of this type when converted to string as it would be
    /// provided by the `__str__` method. If that can't be determined, return `None`.
```

Reply:
I'll do the same for `__repr__`